.github/workflows/version-patch.yml

```
@@ -0,0 +1,154 @@
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Create new release branch
+        run: |
+          git checkout -b ${{ github.event.inputs.tag }}
+      - uses: actions/cache@5a3ec84eff668545956fd18022155c47e93e2684 # v4.2.3
+        id: cache
+        with:
+          path: |
+            node_modules
+            */**/node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+      - name: Cherry pick commit-1
+        if:
+          ${{ github.event.inputs.commit-1 !=
+          '########################################'}}
+        run: |
+          git cherry-pick ${{ github.event.inputs.commit-1 }}
+      - name: Cherry pick commit-2
+        if:
+          ${{ github.event.inputs.commit-2 !=
+          '########################################'}}
+        run: |
+          git cherry-pick ${{ github.event.inputs.commit-2 }}
+      - name: Cherry pick commit-3
+        if:
+          ${{ github.event.inputs.commit-3 !=
+          '########################################'}}
+        run: |
+          git cherry-pick ${{ github.event.inputs.commit-3 }}
+      - name: Cherry pick commit-4
+        if:
+          ${{ github.event.inputs.commit-4 !=
+          '########################################'}}
+        run: |
+          git cherry-pick ${{ github.event.inputs.commit-4 }}
+      - name: Cherry pick commit-5
+        if:
+          ${{ github.event.inputs.commit-5 !=
+          '########################################'}}
+        run: |
+          git cherry-pick ${{ github.event.inputs.commit-5 }}
+      - name: Install dependencies
+        run: yarn install --immutable --immutable-cache --check-cache
+      - name: Update telemetry.yml
+        run: yarn run telemetry:config
+      - name: Create patch release
+        run: |
+          yarn lerna version patch --no-git-tag-version --no-push --yes
+      - name: Create release commit
+        run: |
+          git add -A
+          git commit -m 'chore(release): ${{github.event.inputs.tag}}'
+          git push --set-upstream origin release/${{github.event.inputs.tag}}
+      - name: Log previous 7 commits
+        run: |
+          echo "This should show the new release commit, followed by the cherry-picked commits, followed by the previous release (existing tag), potentially followed by extra unrelated commits if no more than 1 commit was cherry-picked"
+          echo "$(git log -1)"
+          echo "$(git log -2)"
+          echo "$(git log -3)"
+          echo "$(git log -4)"
+          echo "$(git log -5)"
+          echo "$(git log -6)"
+          echo "$(git log -7)"
+      - name: Tag the release commit
+        run: |
+          git tag -a ${{github.event.inputs.tag}} -m '${{github.event.inputs.tag}}'
+      - name: Push the tag upstream
+        run: |
+          git push upstream refs/tags/${{github.event.inputs.tag}}
```

Should the remote be `origin`?

```suggestion
      - name: Tag the release commit
        run: |
          git tag -a ${{github.event.inputs.tag}} -m '${{github.event.inputs.tag}}'
          git push origin refs/tags/${{ github.event.inputs.tag }}
```